SIP-6 Add Film content profile (#151)
viaacode/documentation: docs/diginstroom/sip/2.0/profiles/film.md

---
layout:       default
title:        Film
parent:       Profiles

> grand_parent:  2.0

Author: Moved to SIP 2.1.

nav_order:    4
nav_exclude:  false
---
Editor's Draft
{: .label .label-yellow }
# Profile: Film 

> The film profile supports the ingest of digitised film consisting of one or more image and/or audio reels. 

Reviewer: ```suggestion
The film profile supports the ingest of digitised film stored on one or more image and/or audio reels. 
```

This profile dictates how the media files (in file formats such as ZIP, MOV and DPX), their metadata, and the relationships between them, should be expressed and organized.

> It mainly applies the [DCTERMS metadata schema](https://www.dublincore.org/schemas/xmls/qdc/dcterms.xsd) for descriptive metadata and allows extensions using [Schema.org](https://schema.org), thereby resembling the [Material Artwork profile](https://data.hetarchief.be/id/sip/2.0/material-artwork). 

Reviewer: From 1.2 on, the material artwork profile and the basic profile are aligned in terms of descriptive metadata, so  it's better to refer to the basic profile (that's what material artwork does)

> Its additions lie in the introduction of a separate representation to denote the physical carrier(s) (a so-called 'carrier representation') and custom film-specific metadata (using `<premis:significantProperties>` elements in the package PREMIS file) to describe physical aspects of this/these carrier(s).

Reviewer 2: This sounds like there is an actual representation in the representation folder, which I'm guessing is not?

Author: I have tried to be a bit more specific that this isn't the case.

> This carrier representation was added to facilitate the description of the physical carrier(s), since the PREMIS metadata schema itself doesn't offer this possibility. 

Reviewer: ```suggestion
This carrier representation was added to facilitate the description of the physical carrier(s), since the PREMIS metadata schema itself doesn't offer this possibility directly. 
```

**Permalink:** <https://data.hetarchief.be/id/sip/2.0/film>

## Example Directory structure

```plaintext
root_directory
├── METS.xml
├── metadata
│   ├── descriptive
│   │   └── dc+schema.xml
│   └── preservation
│       └── premis.xml                                  # package PREMIS
│
└── representations
    │
    ├── representation_1
    │   ├── METS.xml
    │   ├── data
    │   │   └── master.zip                              # ZIP containing DPX files
    │   └── metadata
    │       └── preservation
    │           └── premis.xml
    │
    ├── representation_2
    │   ├── METS.xml
    │   ├── data
    │   │   └── mezzanine.mov                           # Playable mezzanine
    │   └── metadata
    │       └── preservation
    │           └── premis.xml
    │
    └── representation_3
        ├── METS.xml
        ├── data
        │   └── QC.dpx                                  # DPX for QC
        └── metadata
            └── preservation
                └── premis.xml
```

## Requirements

### General

- A SIP MUST contain content of exactly one digitised film, consisting of one or more image and/or audio reels.

> - <mark> The content MUST be digitised per reel, i.e. each ZIP and/or MOV and/or DPX file contained in their respective representation directories MUST represent exactly one film consisting of one or more image and/or audio reels.</mark>

Reviewer: I this is correct (check with team digi), but the phrasing is a little weird: we're not steering digitization here, so the sentence before  i.e. is irrelevant in this context and the last part says enough.

I don't know if the following suggestion is still correct:
```suggestion
- Each ZIP, MOV, or DPX file contained in their respective representation directories MUST represent exactly one image or audio reel.
```

- There MUST be exactly one IE present in the SIP, i.e. the digitised film.
- There MUST be preservation metadata at the package level in the `preservation/premis.xml` file.
- There MUST be preservation metadata at the representation level in the respective `preservation/premis.xml` files.
- Preservation metadata in the SIP MUST be limited to the PREMIS metadata schema.
- Fixity MUST be calculated using the MD5 hashing algorithm, thus:
  - The value of element `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm` MUST be set to `MD5`.
  - The value of attribute `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm/@valueURI` MUST be set to `"http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/md5"`.
  - The value of all `//*/@CHECKSUMTYPE` attributes in the `METS.xml` files MUST be set to `MD5`.

> - <mark>There MAY be descriptive metadata at the representation level (e.g. information about the representations, such as a title or a description).</mark>

Reviewer 2: No mention in `general` that package descriptive metadata is mandatory. This is confusing as you do state that preservation metadata is mandatory on package level.

Author: Replaced by a SHOULD.

It states that _preservation_ metadata MUST be present at the package level, while _descriptive metadata about the carrier(s)_ SHOULD be present at the package level. Since both are placed in the same file (package premis.xml) this can be confusing, but it should be correct.

### Package METS

- The `/mets/@TYPE` attribute MUST be set to `Video – File-based and Physical Media`.
- The `csip:CONTENTINFORMATIONTYPE` attribute MUST be set to `OTHER` and the `csip:OTHERCONTENTINFORMATIONTYPE` attribute MUST be set to `https://data.hetarchief.be/id/sip/2.0/film`.
- The `mets/dmdSec/mdRef/@MDTYPE` attribute MUST be set to `OTHER` and the `mets/dmdSec/mdRef/@OTHERMDTYPE` MUST be set to `DC+SCHEMA`.

### Package Descriptive Metadata

- A descriptive metadata file `descriptive/dc+schema.xml` describing the IE  MUST be present at the package level.
- Descriptive metadata in the `descriptive/dc+schema.xml` MUST be limited to the DCTERMS and SCHEMA elements outlined in the [basic profile](https://developer.meemoo.be/docs/diginstroom/sip/2.0/profiles/basic.html#dc-requirements).

> - The DCTERMS and SCHEMA metadata in the `descriptive/dc+schema.xml` file MUST follow the [basic profile requirements](https://developer.meemoo.be/docs/diginstroom/sip/2.0/profiles/basic.html#dc-requirements) regarding the use of elements and attributes.

Reviewer 2: The first line in that link: 
> There MUST NOT be any descriptive metadata at the representation level.

Which isn't the case for this profile.

Author: I eliminated any  mention of descriptive metadata at the representation level since I currently don't know of/see any use case. So this contradiction should be resolved.

In any case, the link that is used has mostly to do with the elements that are outlined in that section and not with the requirements stated in it.

> ### Package Preservation Metadata

Reviewer 2: This section is quite confusing. This might be more logically structured making it more human-readable.

Author: Reformatted this section with some more 'guidance' about how it is structured and a separate intro section for descriptive metadata in the carrier representation (next to a normative summary of requirements as we usually do).

> - If the SIP needs to contain descriptive metadata about the carrier and its reels themselves, the package `preservation/premis.xml` MUST contain a `<premis:object>` for the carrier representation;

Reviewer: These are a bit long for rules. I would add a subsection specifically on describing a carrier with intro paragraph. You can remove the  multiple 'If a carrier representation is present'

- If a carrier representation is present, a structural `<premis:relationship>`  of type 'represents' MUST exist between the `<premis:object>` of the carrier representation and the `<premis:object>` of the intellectual entity (see [Overview of relevant PREMIS relationships]({{ site.baseurl }}{% link docs/diginstroom/sip/2.0/sip_structure/5_structure_package.md %}#premis-relationships) for more information);
- Any descriptive metadata about the physical film's reel(s) MUST be included as part of the carrier representation `<premis:object>` in separate `<premis:significantProperties>` elements;
- If descriptive metadata about the reel(s) is included, the metadata field name MUST be placed in a `<premis:significantPropertiesType>` element with the corresponding metadata field value in a `<premis:significantPropertiesValue>` element (both nested in one of the aforementioned `<premis:significantProperties>` elements);

> - The carrier representation `<premis:object>` MUST contain the carrier type in a `<premis:storageMedium>` element, nested inside a `<premis:storage>` element;

Reviewer: So for each reel we add a `premis:storage` as we discussed?
